src/dialogxml/widgets/control.hpp

```diff
@@ -149,6 +149,7 @@ class cControl {
 	/// @throw xMissingAttr if a required attribute is missing
 	/// @throw xMissingElem if a required attribute is either missing or present in insufficient quantity
 	virtual void validatePostParse(ticpp::Element& who, std::string fname, const std::set<std::string>& attrs, const std::multiset<std::string>& nodes);
+	std::string name;
```

Comment:
I don't like the positioning of this data member in the class. I think it should be placed at the bottom of the class, together with all the other data members. Does it actually need to be protected? If so, I'd put a new protected section just above the private section at the bottom of the class.